### ISSUE TITLE
null-safety support

### ISSUE THREAD

**Issue by avkonst** [westracer/fontify#11]
Hi,

Nice utility. Makes things easier.

I wonder when you plan to update to null-safety? Currently I see it is cross-labelled as not supporting null safety:
```
fontify              ✗0.2.0   ✗0.2.0      -           ✗0.2.0
```

**Comment by westracer** [comment]
Hello. I'll update the package to null-safety as soon as I'll have some spare time (maybe in the next few weeks).

**Comment by avkonst** [comment]
I have checked: all dependencies are null-safety ready. I have attempted automated migration suggestions. It has almost done the job, but unfortunately there were some problems. Sorry, I am not able to submit pull request.

**Comment by westracer** [comment]
Released in 0.3.0-nullsafety.0
